docs/testing.rst

```diff
@@ -381,6 +381,66 @@ functionality, the AioHTTPTestCase is provided::
    Handles executing an asynchronous function, using
    the :attr:`AioHTTPTestCase.loop` of the :class:`AioHTTPTestCase`.
 
+Patching unittest test cases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Patching test cases is tricky, when using python older than 3.8 ``'patch'`` does not behave as it has to.
+We recommend using asynctest_ that provides a patch method that is capable of creating
+a magic mock that is async capable. It can be used with decorator as well as with context manager.::
+
+    from aiohttp.test_utils import AioHTTPTestCase, unittest_run_loop
+    from aiohttp.web_app import Application
+    from aiohttp.web_request import Request
+    from aiohttp.web_response import Response
+    from aiohttp.web_routedef import get
+    from asynctest.mock import patch
```

Comment:
Let's put the third-party in a separate section:
```suggestion
    from asynctest.mock import patch

    from aiohttp.test_utils import AioHTTPTestCase, unittest_run_loop
    from aiohttp.web_app import Application
    from aiohttp.web_request import Request
    from aiohttp.web_response import Response
    from aiohttp.web_routedef import get
```

Reply:
Also maybe improve naming @ `from asynctest.mock import  patch as patch_async`?

Reply:
I don't know about changing the name, people should use that patch anyway since it supports both async and sync code. It is true though that patch_async is more explicit, and "explicit is better than implicit" cit. I would leave it to you to decide since you are one of the maintainers of this library.

Reply:
I think I'd prefer an explicit `patch_async` here as this way it clearly communicates that it's not something that people are used to doing.

Reply:
okay, seems fair to me